lib/package.py

```diff
@@ -56,7 +56,8 @@ def __init__(self, name):
         self.download_source = None
         self.install_dependencies = []
         self.build_dependencies = []
-        self.result_packages = []
+        self.artifacts_dir = os.path.join(
```

Comment:
The class' name is `Package`, so `Package.results` does not make much sense to me. I admit `Package.artifacts` doesn't either. What do you think of `Package.build_results_dir`?